pkg/dataobj/sections/streams/builder.go

```diff
@@ -369,6 +391,10 @@ func encodeColumn(enc *encoder, columnType streamsmd.ColumnType, builder *datase
 		// successfully committed.
 		_ = columnEnc.Discard()
 	}()
+	if len(column.Pages) == 0 {
```

Comment:
columnar.Encoder allows encoding empty columns, so we need an explicit check now. Some tests break without these checks.